Header header
uint8 object_type # Types can be found in object_detection_common/include/DetectionObjects.h
# The following values represent a rect which encloses the object in the image
float64 center_x
float64 center_y
float64 size_x
float64 size_y
int32 img_width
int32 img_height